tests/data/test_config.yml

```diff
@@ -1,6 +1,7 @@
 definitions:
   creds_path: "../../credentials.json"
   token_pickle: "token.pickle"
+  synapse_config: "../../.synapseConfig"
```

Comment:
@linglp could you add a comment here that this is needed only if you have used the Synapse token authentication approach?